### PR TITLE
Help faq

### PR DESCRIPTION
**Pull request by farzanehfar**
1. Help containers changed from custom to react
2. All the containers of Help, have blue Header
3. Added collapse and expand FAQ contents:
** npm install react-native-drop-down-item --save

**** Known Issue====> ** for solving the warning: `useNativeDriver` was not specified. This is a required option and must be explicitly set to `true` ====> 
** https://blog.cpming.top/p/animated-usenativedriver-was-not-specified:

Refer to  Animated doc , with animation types or composition functions, for example, Animated.decay(), Animated.timing(), Animated.spring(), Animated.parallel(), Animated.sequence(), specify  useNativeDriver .
************(Temporary solution)*******Added:  useNativeDriver: true,

Note: If you run the project with debugger mode, It will expand slowly

### CODE REVIEW

File: src/containers/FAQ.js [hunk 1]
```diff
@@ -1,121 +1,145 @@
...
 } from 'react-native/Libraries/NewAppScreen';
 
-import {connect} from 'react-redux';
 
-import {mapDispatchToProps} from '../ducks/actions';
+const IC_ARR_DOWN = require('../icons/ic_arr_down.png');
+const IC_ARR_UP = require('../icons/ic_arr_up.png');
+
+type Props = {};
+export default class App extends Component<Props> {
```
Comment (drewmiley) [inline src/containers/FAQ.js:23]: Can we use Hooks? https://reactjs.org/docs/hooks-intro.html

Comment (drewmiley) [inline src/containers/FAQ.js:23]: This should be called FAQ?

File: src/App.js [hunk 2]
```diff
@@ -28,9 +28,42 @@ const store = configureStore();
 const HelpStack = () => (
   <StackNavigator>
     <StackScreen name="Help" component={Help} />
-    <StackScreen name="FAQ" component={FAQ} />
-    <StackScreen name="AboutDevices" component={AboutDevices} />
-    <StackScreen name="ContactUs" component={ContactUs} />
+    <StackScreen name="FAQ"
+                 component={FAQ}
+                 options={{ title: 'FAQ',
```
Comment (drewmiley) [inline src/App.js:33]: It might be best to draw these repeated styles out?

File: package.json [hunk 3]
```diff
@@ -16,6 +16,9 @@
     "@react-navigation/stack": "^5.3.3",
     "react": "16.11.0",
     "react-native": "0.62.2",
+    "react-native-animatable": "^1.3.3",
```
Comment (drewmiley) [inline package.json:19]: Where is this used? I can't see it in the code.

File: package.json [hunk 4]
```diff
@@ -16,6 +16,9 @@
...
+    "react-native-collapsible": "^1.5.2",
```
Comment (drewmiley) [inline package.json:20]: Where is this used? I can't see it in the code.

File: package.json [hunk 5]
```diff
@@ -16,6 +16,9 @@
...
+    "react-native-drop-down-item": "^1.1.0",
```
Comment (drewmiley) [inline package.json:21]: Why is this not in yarn.lock?

Comment (drewmiley) [reply package.json:21]: Ah, remove package-lock.json ^

File: src/App.js (hunk 2 above)
Comment (farzanehfar) [reply src/App.js:33]: yes, I noticed it when I wrote them but I am going to revise them in another branch if that is fine? in this branch, I already have lots of changes...

**Pull request merged**